package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "reliable-websocket",
+  "version": "0.0.0",
+  "description": "reliable websocket client",
+  "main": "dist/index.js",
```

Comment:
We want to go for es6 modules, so our typescript config should default to es6, and may be create a separate build for cjs.

```json
{
  "main": "dist/index.js",
  "module": "es6/index.js",
  "type": "module",
  "react-native": "src/index.js"
}
```